JellyfinPlayer/VideoPlayer.swift

```diff
@@ -543,7 +542,29 @@ class PlayerViewController: UIViewController, GCKDiscoveryManagerListener, GCKRe
                 .store(in: &cancellables)
         }
     }
-
+    
+    func setupTracksForPreferLanguage() {
```

Comment:
not proper function naming.
"setupTrakcsForPreferredDefaults"

Reply:
tracks?

Reply:
yes :p